### PR TITLE
Improve explainer with info about ndef and security

### PR DESCRIPTION
**Pull request by kenchris**

### CODE REVIEW

File: EXPLAINER.md [hunk 1]
```diff
@@ -58,7 +58,22 @@ Lots of examples can be found here: https://w3c.github.io/web-nfc/#examples
 
 ## Key scenarios
 
-Web NFC works well with other existing APIs
+Among different use-cases, NFC allows users to take their NFC enabled device
+such a a phone and touch NFC tags (sticker, card at the like). The platform,
```
Comment (beaufortfrancois) [inline EXPLAINER.md]: `s/such a/such as/`

Comment (beaufortfrancois) [reply EXPLAINER.md]: `s/card at/card and/`

File: EXPLAINER.md [hunk 2]
```diff
@@ -58,7 +58,22 @@ Lots of examples can be found here: https://w3c.github.io/web-nfc/#examples
...
+or a certain application can then at this moment read the content and write
+new content back.
+
+Web NFC allows to build web experiences that can do the above, thus exchange
+data between the web enabled device and NFC tags.
+
+Data read from NFC tags can come from unknown sources and thus should be
+treated like that by the web developer. This is no different than treating
```
Comment (beaufortfrancois) [inline EXPLAINER.md]: `s/by the web developer/by web developers/`

File: EXPLAINER.md [hunk 3]
```diff
@@ -58,7 +58,22 @@ Lots of examples can be found here: https://w3c.github.io/web-nfc/#examples
...
+user input and data such as JSON, fetched from a remote server.
+
+NDEF signature records can be done to validate that data comes from a trusted
```
Comment (beaufortfrancois) [inline EXPLAINER.md]: `s/can be done/can be used/`

**Review by beaufortfrancois — APPROVED** [review]
LGTM with nits.

File: EXPLAINER.md (hunk 3 above)
Comment (zolkis) [inline EXPLAINER.md]: It's not validation that signature records enable, but content integrity (i.e. no one has tampered with the signed content).

Comment (kenchris) [reply EXPLAINER.md]: Well you can encode id right...

Comment (kenchris) [reply EXPLAINER.md]: How would you rewrite it?

Comment (zolkis) [reply EXPLAINER.md]: It's also authentication, as there is a chain of certificates in a signature.

The Signature RTD spec says 

> It provides users with the possibility of verifying the authenticity and integrity of data within the NDEF message.

File: EXPLAINER.md [hunk 4]
```diff
@@ -58,7 +58,22 @@ Lots of examples can be found here: https://w3c.github.io/web-nfc/#examples
...
+source and it is possible for web sites to ignore all other records.
```
Comment (zolkis) [inline EXPLAINER.md]: > it is possible for web sites to ignore all other records

An explicit formulation like "all non-signed records" (or something like that) would be more specific.

File: EXPLAINER.md [hunk 5]
```diff
@@ -169,9 +184,10 @@ document.onvisibilitychange = event => {
 ## Detailed design discussion
 
 ### Low level API vs standardized NDEF records only
-NDEF comes with a set of standard records with data encoded in particular ways,
-like text which also encode the encoding and language name, and even
-smart-poster, which can contain icons and descriptions in multiple languages.
+NDEF (NFC Data Exchange Format) comes with a set of standard records with
+data encoded in particular ways, like text which also encode the encoding
+and language name, and even smart-poster, which can contain icons and
+descriptions in multiple languages.
```
Comment (zolkis) [inline EXPLAINER.md:191]: Mention media types? Though not much used. URL records are among the most used, though.

**Review by zolkis — APPROVED** [review]
Nitpicking suggestions made. Thanks!

File: EXPLAINER.md [hunk 6]
```diff
@@ -59,19 +59,20 @@ Lots of examples can be found here: https://w3c.github.io/web-nfc/#examples
...
+treated like that by web developers. This is no different than treating
 user input and data such as JSON, fetched from a remote server.
 
-NDEF signature records can be done to validate that data comes from a trusted
-source and it is possible for web sites to ignore all other records.
+The NDEF signature records, gives the web site the possibility of verifying
+the authenticity and integrity of data within the NDEF message and only deal
+with data it trusts.
 
 Aditionally, Web NFC integrates well with other existing APIs:
```
Comment (zolkis) [inline EXPLAINER.md]: Additionally

**Review by zolkis — APPROVED** [review]

**Pull request merged**